HW_1/task_2.py

```diff
@@ -0,0 +1,14 @@
+"""
+Докажите, что выражение ¬(X ⋁ Y ⋁ Z) = ¬X ⋀ ¬Y ⋀ ¬Z истинно для всех значений предикат.
+
+Вывод: единственное значение типа bool (True либо False)
+"""
+
+x = int(input("Введите x: "))
+y = int(input("Введите y: "))
+z = int(input("Введите z: "))
+
+if not (x or y or z) == (not x and not y and not z):
+    print(bool(1))
+else:
+    print(bool(0))
```

Comment:
```suggestion
    print(True)
else:
    print(False)
```
Ваше решение не полное, вы проверяете только одну комбинацию значений предикат, а их всего может быть 8. Это вам небольшая подсказка